### ISSUE TITLE
String lengths for mssql

### ISSUE THREAD

**Issue by Benjathing** [sequelize/sequelize-auto#219]
Testing `sequelize-auto` with a model -> database -> auto:

    export default function(sequelize, DataTypes) {
        return sequelize.define('test', {
            title: DataTypes.STRING(1000),
      });
    }

generated a table with:
`title (nvarchar(1000), null)`

However, on running sequelize-auto against this database, it genereated only:

    module.exports = function(sequelize, DataTypes) {
      return sequelize.define('test', {
        title: {
          type: DataTypes.STRING,
          allowNull: true
        }
    }

What happened to the string length?

**Comment by irwin-r** [comment]
Hello,

I'm still very new to Sequelize and figuring it out myself, so I can't advise on how to best fix this issue.
I have, however, made a few changes to `sequelize-auto` to add `STRING`/`CHAR `lengths which will hopefully shed some insight into the issue for a maintainer to actually use.

- The crux of the issues lies in Sequelize's **DESCRIBE** `QueryType`, which seems to shed the `Length` column off the `describeTableQuery` response.  
- I didn't have the the energy to explore how/why the **DESCRIBE** `QueryType` does this, so to make things work for myself,  I used the **SELECT** `QueryType` instead.
- I then modified the `isDescribeQuery` portion of `formatResults` to re-add the size suffix to the `type` attribute in an event where a length was specified.

Hope this helps :)

On a somewhat unrelated note, `SMALLDATETIME` and `MONEY` aren't currently acknowledged by Sequelize.  I've currently just routed them to `DATE`/`DECIMAL` respectively, but I think that recognising/adding these types would be a welcome addition.
